### ISSUE TITLE
hexo-math不能引用公式

### ISSUE THREAD

**Issue by swordspoet** [hexojs/hexo-math#34]
我在mathjax.js中配置如下：
```
<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        TeX: {equationNumbers: {autoNumber: ["AMS"], useLabelIds: true}},
        "HTML-CSS": {linebreaks: {automatic: true}},
        SVG: {linebreaks: {automatic: true}}
    });
</script>
```

但\label{id}并不能起到引用公式的作用？

这是bug吗？还是我没有更新的缘故？

**Comment by stevenjoezhang** [comment]
This plugin is being remade. There is a discussion about it: #130
I'm closing this issue. Please join the discussion there :)
